test/parallel/test-zlib-close-after-write.js

```diff
@@ -23,7 +23,7 @@
 const common = require('../common');
 const zlib = require('zlib');
 
-zlib.gzip('hello', common.mustCall(function(err, out) {
+zlib.gzip('hello', common.mustCall((err, out)=> {
```

Comment:
Nit: space is required before arrow to pass the lint test
https://travis-ci.com/nodejs/node/jobs/159977196

Reply:
I didn't know that, I've pushed again with the required changes, thanks!